src/handlers/tests/user-identity-handler.test.ts

```diff
@@ -0,0 +1,19 @@
+import { APIGatewayProxyEvent } from "aws-lambda";
+import { handler } from "../user-identity-handler";
+import { UserIdentityInput } from "../../types/interfaces";
+
+describe("user-identity-handler tests", () => {
+  it("should return Success", async () => {
+    const testObject: UserIdentityInput = {
+      govukSigninJourneyId: "j8mMnXW_rP6JqNXBKKf8xsGXttk121",
+      vtr: ["P1", "P3"],
```

Comment:
I know it doesn't really matter in context of this test, but this isn't a realistic `vtr` ... `["P1", "P2"]` or `["P2", "P3"]` or more realistic.